clinica/pipelines/t1_linear/t1_linear_cli.py

```diff
@@ -0,0 +1,93 @@
+        clinica_comp.add_argument("bids_directory",
+                                  help='Path to the BIDS directory.')
+        clinica_comp.add_argument("caps_directory",
+                                  help='Path to the CAPS directory.')
+
+        # group_id can be used by certain pipelines when some operations are performed at the group level
+        # (for example, generation of a template in pipeline t1-volume)
+        # clinica_comp.add_argument("group_id",
+        #                          help='User-defined identifier for the provided group of subjects.')
+
+        # Clinica standard arguments
+        clinica_opt = self._args.add_argument_group(PIPELINE_CATEGORIES['CLINICA_OPTIONAL'])
+        clinica_opt.add_argument("-tsv", "--subjects_sessions_tsv",
+                                 help='TSV file containing a list of subjects with their sessions.')
+        clinica_opt.add_argument("-wd", "--working_directory",
+                                 help='Temporary directory to store pipelines intermediate results.')
+        clinica_opt.add_argument("-np", "--n_procs",
+                                 metavar=('N'), type=int,
+                                 help='Number of cores used to run in parallel.')
+
+        # Add your own pipeline command line arguments here to be used in the
+        # method below. Example below:
+        # optional = self._args.add_argument_group(PIPELINE_CATEGORIES['OPTIONAL'])
+        # optional.add_argument("-rt", "--ref_template",
+        #                      help='Reference template for registration.')
+
+        # Add advanced arguments
+        # advanced = self._args.add_argument_group(PIPELINE_CATEGORIES['ADVANCED'])
+        # advanced.add_argument("-aa", "--advanced_arg",
+        #                       help='Your advanced argument.')
+        # self.add_clinica_standard_arguments(add_overwrite_flag=True)
+
+    def run_command(self, args):
+        """
+        """
+
+        from tempfile import mkdtemp
+        from .t1_linear_pipeline import T1Linear
+
+        parameters = {
+            #'ref_template'        : args.ref_template or 'Reference Template'
+        }
+
+        # Most of the time, you will want to instantiate your pipeline with a
+        # BIDS and CAPS directory as inputs:
+        pipeline = T1Linear(
+             bids_directory=self.absolute_path(args.bids_directory),
+             caps_directory=self.absolute_path(args.caps_directory),
+             tsv_file=self.absolute_path(args.subjects_sessions_tsv),
+             parameters=parameters,
+             name=self.name
+             )
+        if args.working_directory is None:
+            args.working_directory = mkdtemp()
+        pipeline.base_dir = self.absolute_path(args.working_directory)
+        if args.n_procs:
+            pipeline.run(plugin='MultiProc',
+                         plugin_args={'n_procs': args.n_procs})
+        else:
+            pipeline.run()
```

Comment:
I think the Jinja2 was not updated when you generated the skeleton. The `run_command` should look like:

```python
    def run_command(self, args):
        """Run the pipeline with defined args."""
        from networkx import Graph
        from .t1_volume_tissue_segmentation_pipeline import T1Linear
        from clinica.utils.ux import print_end_pipeline, print_crash_files_and_exit
    
        parameters = {
            #'ref_template'        : args.ref_template or 'Reference Template'
        }
        pipeline = T1Linear(
            bids_directory=self.absolute_path(args.bids_directory),
            caps_directory=self.absolute_path(args.caps_directory),
            tsv_file=self.absolute_path(args.subjects_sessions_tsv),
            base_dir=self.absolute_path(args.working_directory),
            parameters=parameters,
            name=self.name
        )
    
        if args.n_procs:
            exec_pipeline = pipeline.run(plugin='MultiProc',
                                         plugin_args={'n_procs': args.n_procs})
        else:
            exec_pipeline = pipeline.run()
    
        if isinstance(exec_pipeline, Graph):
            print_end_pipeline(self.name, pipeline.base_dir, pipeline.base_dir_was_specified)
        else:
            print_crash_files_and_exit(args.logname, pipeline.base_dir)
```